docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -373,14 +377,18 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Job ID: c8c5s4819ieg4tnjoo0g\n",
-      "{'quasi_dists': [{'49': 0.0224609375, '32': 0.013671875, '35': 0.0244140625, '1': 0.017578125, '55': 0.1279296875, '63': 0.517578125, '48': 0.0390625, '51': 0.05859375, '39': 0.0537109375, '0': 0.0146484375, '7': 0.060546875, '3': 0.03125, '33': 0.0185546875}], 'shots': 1024}\n"
+      "Job ID: c8nohfps9rvto959da30\n",
+      "{'quasi_dists': [{'000001': 0.015625, '100011': 0.0361328125, '110001': 0.0380859375, '100001': 0.01953125, '110000': 0.02734375, '110011': 0.0703125, '111111': 0.4716796875, '110111': 0.130859375, '100000': 0.01171875, '000011': 0.033203125, '000000': 0.0185546875, '000111': 0.0576171875, '100111': 0.0693359375}], 'metadata': [{'header_metadata': None, 'shots': 1024}]}\n",
+      "<IBMBackend('ibmq_qasm_simulator')>\n"
      ]
     }
    ],
    "source": [
     "# Specify the program inputs here.\n",
-    "program_inputs = {\"circuits\": qc}\n",
+    "program_inputs = {\n",
+    "    \"circuits\": qc,\n",
+    "    \"circuit_indices\": [0],\n",
```

Comment:
I don't really understand the value of this parameter and why it's mandatory for this use-case. But given that my context is limited and the decision appears to be fixed already, I won't block it here :)

Reply:
We couldn't settle on a sensible default for circuit_indices when multiple circuits are passed, so this was made mandatory just to be consistent with estimator.